bitnami/postgresql/templates/primary/statefulset.yaml

```diff
@@ -230,7 +230,7 @@ spec:
           {{- end }}
             {{- if .Values.auth.usePasswordFiles }}
             - name: POSTGRES_PASSWORD_FILE
-              value: {{ printf "/opt/bitnami/postgresql/secrets/%s" (ternary "password" "postgres-password" (and (not (empty $customUser)) (ne $customUser "postgres"))) }}
+              value: {{ printf "/opt/bitnami/postgresql/secrets/%s" (include "postgresql.userPasswordKey" .) }}
```

Comment:
Note to reviewers: "postgresql.userPasswordKey" has the logic from the `ternary` statement that is removed.